src/workflows-app/components/LeftNavigationPanel.ts

```diff
@@ -0,0 +1,156 @@
+            title: span({ style: { fontSize: 15, color: colors.accent() } }, ['Find & add workflows']),
+            initialOpenState: true,
+            titleFirst: true,
+            summaryStyle: { padding: '1rem 1rem 1rem 1.5rem' },
+          },
+          [
+            div(
+              {
+                style: { alignItems: 'center', flexDirection: 'column', paddingLeft: '2rem' },
+                role: 'list',
+              },
+              [
+                _.map(([subHeaderKey, subHeaderName]) => {
+                  const isActive = isSubHeaderActive(subHeaderKey);
+                  return h(
+                    Clickable,
+                    {
+                      'aria-label': `${subHeaderKey}-header-button`,
+                      style: { color: isActive ? colors.accent(1.1) : colors.accent(), fontSize: 16 },
+                      onClick: () => setSelectedSubHeader(subHeaderKey),
+                      hover: Style.navList.itemHover(isActive),
+                      'aria-current': isActive,
+                      key: subHeaderKey,
+                    },
+                    [
+                      h(ListItem, {
+                        title: subHeaderName,
+                        style: { borderBottom: `0.5px solid ${colors.dark(0.2)}` },
+                      }),
+                    ]
+                  );
+                }, Object.entries(findAndAddSubheadersMap)),
+                div(
+                  {
+                    style: {
+                      marginRight: '3em',
+                      marginTop: '1.5em',
+                      display: 'flex',
+                      alignItems: 'center',
+                      flex: 'none',
+                      backgroundColor: colors.accent(0.06),
+                      boxShadow: '0 2px 5px 0 rgba(0,0,0,0.35), 0 3px 2px 0 rgba(0,0,0,0.12)',
+                      paddingTop: '0.5em',
+                      paddingBottom: '0.5em',
+                      paddingLeft: '0.75em',
+                      paddingRight: '0.75em',
+                      borderRadius: '15px',
+                    },
+                  },
+                  [
+                    h(
+                      Clickable,
+                      {
+                        role: 'list',
+                        href: `${
+                          getConfig().dockstoreUrlRoot
+                        }/search?_type=workflow&descriptorType=WDL&searchMode=files`,
+                      },
+                      [
+                        div({ style: { fontWeight: 'bold' } }, ['Dockstore  ', icon('export')]),
+                        div([
+                          'Browse WDL workflows in Dockstore, an open platform used by the GA4GH for sharing Docker-based workflows.',
+                        ]),
+                      ]
+                    ),
+                  ]
+                ),
+              ]
+            ),
+          ]
+        ),
+      ]),
+      h(HelpfulLinksBox, {
```

Comment:
Based on the mockups we should fix the position of this box at the bottom of left panel with a "line" drawn above it. Currently it is floating when the page size is changed 🤔 

https://github.com/DataBiosphere/terra-ui/assets/16748522/81591148-8faf-4bc9-825c-0296a6b338cb